spacy/lang/en/stop_words.py

```diff
@@ -66,7 +66,10 @@
 whither who whoever whole whom whose why will with within without would
 
 yet you your yours yourself yourselves
-
-'d 'll 'm 're 's 've
 """.split()
 )
+
+for hyphen in ["'", "`", "‘", "´", "’"]:
+    for stopword in u"n't 'd 'll 'm 're 's 've".split():
```

Comment:
```suggestion
    for stopword in ["n't", "'d", "'ll", "'m", "'re", "'s", "'ve"]:
```

I'd actually write this as a list here, because it's more straightforward. The `split` syntax is easier for long lists like the stop words, but inline, I think a real list is easier to read. Also, no need for the `u` – we have the unicode declarations at the top 🎉

Reply:
Yea, I forgot to add that declaration initially and ran into the unit tests for Python 2.7 ofcourse. Wonder how many more times I'm going to make the same mistake ;-)